### DISCUSSION TITLE
请问如何获取qimei36

### DISCUSSION THREAD

**Discussion by guogai114514** [Mrs4s/go-cqhttp#2426] [Q&A]
qsign的register需要qiemei36，但不知道怎么获取，有大佬出个计算公式吗qwq
谢谢（

**Comment by starpickuper** [comment]
同求，tx爬蟲也需要

**Reply by WangHansch** [reply]
这么久了，层主也许已经找了方法。可以通过抓包工具fidder去找，看网页向服务器中发送的post请求中所含的该数据项。示例为fiddler抓包 腾讯新闻 娱乐类目时所截取。
![屏幕截图 2024-05-19 032953](https://github.com/Mrs4s/go-cqhttp/assets/152622086/48ab013e-1fe7-4c8d-a1be-fcfcafe17a16)
